testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -243,6 +243,26 @@ def test_check_coordinate_limits_max(self):
         assert_raises(ValueError, u.atoms.write, self.outfile)
         del u
 
+    @attr('issue')
+    def test_check_header_title_multiframe(self):
+        """Check whether HEADER and TITLE are written just once in a multi-
+        frame PDB file (Issue 741)"""
+        u = mda.Universe(PSF, DCD, permissive=True)
+        protein = u.select_atoms("protein and name CA")
+        pdb = mda.Writer(self.outfile, multiframe=True)
+        for ts in u.trajectory[:5]:
+            pdb.write(u)
+        pdb.close()
+        u2 = mda.Universe(self.outfile)
+        if u2.trajectory.header:
+            assert_equal(len(u2.trajectory.header),
+                        1,
+                        err_msg='There should be only one HEADER')
+        if u2.trajectory.title:
```

Comment:
Think about what you are testing, and why you are testing it. On a general note, there are a few question you have to answer every time you write a test:

* what is the expected behaviour?
* how can thing go wrong?
* does your test pass if you get the expected behaviour?
* does your test fail if the behaviour is wrong?